tests/base/src/test/java/org/keycloak/tests/tracing/TracingProviderTest.java

```diff
@@ -1,105 +1,44 @@
-import org.jboss.arquillian.container.test.api.ContainerController;
-import org.jboss.arquillian.test.api.ArquillianResource;
-import org.junit.Before;
-import org.junit.FixMethodOrder;
-import org.junit.Test;
-import org.junit.runners.MethodSorters;
-import org.keycloak.representations.idm.RealmRepresentation;
-import org.keycloak.testsuite.AbstractTestRealmKeycloakTest;
-import org.keycloak.testsuite.arquillian.annotation.AuthServerContainerExclude;
-import org.keycloak.testsuite.arquillian.containers.AbstractQuarkusDeployableContainer;
-import org.keycloak.tracing.NoopTracingProvider;
+import org.junit.jupiter.api.Test;
+import org.keycloak.testframework.annotations.KeycloakIntegrationTest;
+import org.keycloak.testframework.remote.runonserver.InjectRunOnServer;
+import org.keycloak.testframework.remote.runonserver.RunOnServerClient;
+import org.keycloak.testframework.server.KeycloakServerConfig;
+import org.keycloak.testframework.server.KeycloakServerConfigBuilder;
 import org.keycloak.tracing.TracingProvider;
 
 import java.io.Serializable;
-import java.util.List;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.emptyOrNullString;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.notNullValue;
 
-@AuthServerContainerExclude(value = AuthServerContainerExclude.AuthServer.UNDERTOW)
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
-public class OTelTracingProviderTest extends AbstractTestRealmKeycloakTest {
-
-    @ArquillianResource
-    protected ContainerController controller;
-
-    private static boolean initialized;
-
-    @Override
-    public void configureTestRealm(RealmRepresentation testRealm) {
-    }
-
-    @Before
-    public void setContainer() {
-        assertThat(suiteContext.getAuthServerInfo().isQuarkus(), is(true));
-
-        if (!initialized) {
-            startContainer();
-            initialized = true;
-        }
-    }
+@KeycloakIntegrationTest(config = TracingProviderTest.ServerConfigWithTracing.class)
+public class TracingProviderTest {
 
-    @Test
-    // reset configuration and restart server as the last step of tests execution
-    public void zzzzAfterClass() {
-        try {
-            var container = (AbstractQuarkusDeployableContainer) suiteContext.getAuthServerInfo().getArquillianContainer().getDeployableContainer();
-            container.resetConfiguration();
-            container.restartServer();
-        } catch (Exception e) {
-            throw new RuntimeException(e);
+    public static class ServerConfigWithTracing implements KeycloakServerConfig {
+        @Override
+        public KeycloakServerConfigBuilder configure(KeycloakServerConfigBuilder config) {
+            return config
+                    .option("tracing-enabled", "true")
+                    .option("log-level", "INFO,org.keycloak.quarkus.runtime.tracing:debug");
         }
```

Comment:
By convention, we put the class with server config to the bottom of the test class